enhancements/sig-architecture/58-addon-placement-annotation/README.md

```diff
@@ -0,0 +1,65 @@
+# The Placement Annotation of Addon
```

Comment:
Minor nit. Can we use a different word instead of `Placement`? We have the Placement API and this enhancement is talking about something that is not related to that.

Reply:
change to `The Addon Node Selector and Toleration Annotation`